peekingduck/pipeline/nodes/input/recorded.py

```diff
@@ -83,7 +103,10 @@ def run(self, inputs: Dict[str, Any]) -> Dict[str, Any]:
             self.tens_counter += 10
 
         if self.file_end:
-            self.logger.info(f"Completed processing file: {self._file_name}")
+            pct_complete = round(100 * self.frame_counter / self.videocap.frame_count)
+            self.logger.info(
+                f"Processed {self._file_name}: #frames={self.frame_counter}, done={pct_complete}%"
+            )
             self._get_next_input()
```

Comment:
Actually I put `pct_complete` there precisely to identify and weed out such cases. 😅
What we ideally want is `100%` because this would mean a) no frames were lost (else would be lower than 100%) and b) no extra frames were processed (but with `buffer_frames`, this may not hold true).
One option is to set this as a debug output instead of a standard info output.

Reply:
Okay, I see. Perhaps we can set it as a debug output instead?

Reply:
Done, changed to debug output.